README.md

```diff
@@ -70,18 +70,18 @@ Supported Compilers
+| TBB                                                                            | :white_check_mark:                        | :white_check_mark:                        | :white_check_mark:                        | :white_check_mark:                   | :white_check_mark:     | :white_check_mark:      | :white_check_mark:                              | :white_check_mark:                               | :white_check_mark:                       | :white_check_mark:                        | :white_check_mark:                   | :x:                                | :white_check_mark: | :white_check_mark:                                    | :x:                                       | :x:                                  |
+| CUDA (nvcc)                                                                    | :white_check_mark: <br/> (CUDA 11.0-11.6) | :white_check_mark: <br/> (CUDA 11.0-11.6) | :white_check_mark: <br/> (CUDA 11.0-11.6) | :white_check_mark: <br/> (CUDA 11.6) | :x:                    | :x:                     | :white_check_mark: <br/> (CUDA 11.0-11.2; 11.6) | :white_check_mark: <br/> (CUDA 11.1, 11.2, 11.6) | :white_check_mark: <br/> (CUDA 11.6)     | :white_check_mark: <br/> (CUDA 11.6)      | :white_check_mark: <br/> (CUDA 11.6) | :x:                                | :x:                | :x:                                                   | :white_check_mark: <br/> (CUDA 11.2-11.6) | :white_check_mark: <br/> (CUDA 11.6) |
+| CUDA (clang)                                                                   | -                                         | -                                         | -                                         | -                                    | -                      | -                       | -                                               | :x:                                              | :white_check_mark: <br/> (CUDA 9.2-10.1) | :white_check_mark: <br/> (CUDA 10.0-10.2) | -                                    | :x:                                | :x:                | -                                                     | -                                         | -                                    |
+| [HIP](https://alpaka.readthedocs.io/en/latest/install/HIP.html) (clang)        | :x:                                       | :x:                                       | :x:                                       | :x:                                  | :x:                    | :x:                     | :x:                                             | :x:                                              | :x:                                      | :white_check_mark: (HIP 4.2)              | :white_check_mark: (HIP 4.3 - 4.5)   | :white_check_mark: (HIP 5.0 - 5.1) | :x:                | -                                                     | -                                         | -                                    |
```

Comment:
Version 5.2 is still missing here. Since AMD doesn't publish the LLVM version they use as basis I need to wait for the output of `clang --version` on the CI. I will amend the commit once I have the information.

Reply:
Done. ROCm 5.2 is still using llvm-14.